charts/alertmanager/templates/statefulset.yaml

```diff
@@ -97,9 +97,20 @@ spec:
         - name: {{ .Chart.Name }}-{{ .Values.configmapReload.name }}
           image: "{{ .Values.configmapReload.image.repository }}:{{ .Values.configmapReload.image.tag }}"
           imagePullPolicy: "{{ .Values.configmapReload.image.pullPolicy }}"
+          {{- with .Values.configmapReload.extraEnv }}
+          env:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
           args:
+            {{- if not (hasKey .Values.configmapReload.extraArgs "config-file") }}
```

Comment:
Config-file vs watched-dir? Why's that again?

Reply:
Hi! (Sorry, not at my work computer so replying from another account). The config-reloader can watch individual files, and then populate environment variables and output to a new file. I just wanted more flexibility about the options passed to the config reloader in general.

Reply:
np, but is there a use case where user passes a `watched-dir` key to `extraArgs` ? if so how would that be handled here

Reply:
Yeah, that wouldn't work. I can make a change to check and see if that value has been set as well.

Reply:
Thanks for waiting, just made the change to allow overriding `watched-dir`!